drools-game-engine-core-api/src/main/java/org/drools/game/core/api/StoreFacts.java

```diff
@@ -0,0 +1,9 @@
+package org.drools.game.core.api;
+
+import java.util.List;
+
+public interface StoreFacts {
```

Comment:
@Salaboy I think that this interface is not necessary since this method is already exposed in GameConfiguration. If I am right, should be an instance of GameConfiguration injected in GameServiceImpl?